### PR TITLE
Revising README.md

### PR DESCRIPTION
**Pull request by TheRealPear**
Images and edits from other contributors are welcome. We need to make it easy for mapmakers by having all the information and resources available in one place.

### CODE REVIEW

**Comment by CrazyisCreeps** [comment]
idk if I'd bother linking the old docs at docs.oc.tc, as they aren't up to date and can show modules no longer support and/or miss out on new modules that have been added

File: README.md [hunk 1]
```diff
@@ -6,30 +6,30 @@ This repository contains public maps and active rotation list that are found on
...
+- This can be achieved with third-party programs such as [MCASelector](https://github.com/Querz/mcaselector), [MCEdit-Unified](https://github.com/Podshot/MCEdit-Unified), or [MCEdit 2.0](https://github.com/mcedit/mcedit2).  
+- **Note:** When using MCASelector, please make sure you select the `region` folder inside a world folder to edit a world, not the entire world folder.  
+- After pruning the map, please make sure that you only include the required world files (`region/*`, `level.dat`, `map.png`, `map.xml`, ).  
+  - For more information about cleaning the world files, [view this page on PGM Documentation](https://pgm.dev/docs/guides/packaging/cleaning-files).  
 
 ### XML Creation
-  * All maps must include a valid `map.xml` to function properly with PGM. The map protocol must be `1.4.0` or above.
-  * The map version **must** follow the [Semantic Versioning](https://semver.org/) standard.
-  * Refer to [Additional Resources](#additional-resources) on following proper XML format.
+- All maps must include a valid `map.xml` to function properly with PGM. The map protocol must be `1.4.0` or above.
```
Comment (TheRealPear) [inline README.md:22]: I thought going above proto version 1.4.0 was a 1.9 PGM thing?

Comment (TBG1000) [reply README.md:22]: No, a good chunk of features supported by higher protocols have already been introduced to current PGM and work without trouble. Some OCC’s community-submitted maps (and some maps located at their new PublicMaps repo) are already using these protocols.

**Comment by TheRealPear** [comment]
How would you guys feel about crediting ResourcePile and OCC in the acknowledgements section? Some of our maps come from them and we only had a small discussion in #mapmaking, but nothing more.

**Pull request merged**